### ISSUE TITLE
hello function not working?

### ISSUE THREAD

**Issue by ahmedelgabri** [Jakobo/eyeglass-sample#1]
I was playing with eyglass, so I created this example but it seems the `hello($name)` function is not being processed.

The `index.scss` file

``` sass
@import 'eyeglass-sample';

.foo {
  content: hello("name");
}
```

`index.js` file

``` js
var sass = require('node-sass');
var Eyeglass = require('eyeglass');
var sassOptions = {
  file: './index.scss',
  outFile: './index.css'
};
var eyeglass = new Eyeglass(sassOptions);

sass.render(eyeglass.sassOptions(), function(err, res){
  if (err) console.log(err);
  console.log(res.css.toString());
});
```

Output `index.css` file

``` css
.sassy {
  color: #c69; }

.foo {
  content: hello("name"); }
```

Not sure if I'm missing something here cause the `.sassy` class is exported which means the `@import` is working but I dunno why `hello()` is not working, I also added [eyeglass-math](https://github.com/sass-eyeglass/eyeglass-math) to see if there is something wrong with the way I compile. But it did work perfectly.

Input:

``` sass
@import 'eyeglass-sample';
@import 'math';

.foo {
  content: hello("name");
  content: $PI;
  content: sqrt(144);
}
```

Output

``` css
.sassy {
  color: #c69; }

.foo {
  content: hello("name");
  content: 3.14159; 
  content: 12;}
```

**Comment by franzheidl** [comment]
Same here. 
Importing the module works ~~(although it seems I have to use `@import "module-name/module-name"`???)~~, but the functions as defined in eyeglass-exports.js don't seem to be executed:

SCSS:

```
  @import "eyeglass-sample";

  .test {
    @extend .sassy;  // extend the sample class from eyeglass-sample to test whether it is found
  }

  .hello {
    content: hello("Chris");
  }
```

CSS:

```
  .sassy, .test {
    color: #c69;   // extending works!
  }

  .test {
    content: hello("Chris");  // Huh?!?
  }

```

**Comment by thecodedrift** [comment]
I'll take a look at this today and see if we messed something up in the sample module (likely since eyeglass-math works as expected)

Do you mind sharing your `package.json` as well? I did notice a problem where in eyeglass 2, the `devDependencies` are not being looked at consistently. If I force includeDevDependencies to `true`, then it works, but the side effects are unknown.

**Comment by ahmedelgabri** [comment]
@Jakobo here is the `package.json` and I think you are right about `devDependencies`. I think I added it as a dev dependency by mistake and that's why it's not working.

``` json
{
  "name": "eyeglass-test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "MIT",
  "devDependencies": {
    "eyeglass": "^0.2.0",
    "eyeglass-sample": "0.0.2",
    "node-sass": "^3.1.2"
  },
  "dependencies": {
    "eyeglass-math": "^1.0.1"
  }
}
```

But after moving it inside `dependencies` now this is the output 

``` css
.sassy {
  color: #c69; }

.foo {
  content: "Why hello, [object SassString]";
  content: 12;
  content: 3.14159; }
```

Shouldn't this be `Why hello, name` instead?

**Comment by franzheidl** [comment]
I have eyeglass and eyeglass-sample in my devDependencies, too. 
Maybe I'm not getting this right, but the above sounds as if eyeglass is supposed to be used as a dependency only?
I'd expect eyeglass and any eyeglass-modules to work identically both as a dependency as well as a devDependency.

**Comment by thecodedrift** [comment]
https://github.com/sass-eyeglass/eyeglass/pull/41 found the problem in eyeglass. We'll get it fixed @franzheidl @ahmedelgabri. For now, in `dependencies` will work until I can land the fix back into eyeglass.

Thanks for your patience all!

**Comment by ahmedelgabri** [comment]
Thanks @Jakobo :+1:

**Comment by eoneill** [comment]
This was resolved with https://github.com/sass-eyeglass/eyeglass/pull/41 and released in `eyeglass@0.3.0` and later.
